Libs/DICOM/Widgets/ctkDICOMServerNodeWidget2.h

```diff
@@ -92,7 +92,7 @@ class CTK_DICOM_WIDGETS_EXPORT ctkDICOMServerNodeWidget2 : public QWidget
   Q_INVOKABLE int serversCount();
   Q_INVOKABLE ctkDICOMServer* getNthServer(int id);
   Q_INVOKABLE ctkDICOMServer* getServer(const QString& connectionName);
-  Q_INVOKABLE void addServer(ctkDICOMServer* server);
+  Q_INVOKABLE int addServer(ctkDICOMServer* server);
   Q_INVOKABLE void removeServer(const QString& connectionName);
   Q_INVOKABLE void removeNthServer(int id);
```

Comment:
`nth` is not Qt style

```suggestion
  Q_INVOKABLE void removeServer(int id);
```